tools/utils/sourceList.py

```diff
@@ -15,7 +15,7 @@ def list_typesystem_cpp_sources(typesystem, out):
     sources = [f"{package.lower()}_module_wrapper.cpp"]
     sources.extend([f"{typename.lower()}_wrapper.cpp" for typename in types])
 
-    return [os.path.normpath(os.path.join(out, package, f)) for f in sources]
+    return [os.path.normpath(os.path.join(out, package, f)).replace("\\", "/") for f in sources]
```

Comment:
Why do you need this? I was able to build with VS code w/o it. I'm assuming you are trying to deal with Windows native paths, but it's not clear to me why you are having to deal with those here. Where are the backslashes coming from?

Reply:
This is the error I get when builiding with VScode : 

```
[cmake] CMake Error at Engine/CMakeLists.txt:69 (add_library):
[cmake]   Syntax error in cmake code when parsing string
[cmake] 
[cmake]     D:\Development\sources\Natron\BUILD\Engine\Qt5\NatronEngine\natronengine_module_wrapper.cpp
[cmake] 
[cmake]   Invalid character escape '\D'.
```
Note that there's no build issue with a MINGW64 bash shell

Reply:
Hmm... I wonder if there is a mismatch between which python and/or cmake is being invoked. Maybe adding "COMMAND_ECHO STDERR" to the execute_process call that runs this might provide some clues. It also might be good to look up earlier in the build output and make sure that proper python and cmake binaries are being used.